nexus/external-api/src/lib.rs

```diff
@@ -1166,6 +1166,18 @@ pub trait NexusExternalApi {
         path_params: Path<params::InstancePath>,
     ) -> Result<HttpResponseAccepted<Instance>, HttpError>;
 
+    /// Stop instance, violently.
```

Comment:
```suggestion
        /// Terminate instance
        ///
        /// Immediately halts a running instance by rudely terminating its
        /// virtual machine process. This immediately moves the instance to the
        /// "stopped" state without transitioning through the "stopping" state.
        /// This operation can be used to recover an instance that is not
        /// responding to requests to stop issued through the instance stop API.
```

The difference between this and regular instance stop is subtle enough that I'm inclined to wordsmith the documentation a bit. The tricky bit is that the two operations look the same to the guest OS, so we can't pitch this as "stop tries to shut down the guest, terminate is a power pull"--it's really more "stop is a power pull, terminate blows away the current running VM state so you can try again from Stopped." Hopefully this will get readers pointed in the right direction.